http_server_check/check.py

```diff
@@ -23,7 +23,7 @@ def generate_message(min_len=32, max_len=64):
 class TestException(Exception):
     pass
 
-def handle_pexpect(child_process, processes_to_terminate, expect_string, output_buffer, step, timeout=1):
+def handle_pexpect(child_process, processes_to_terminate, expect_string, output_buffer, step, timeout=3):
```

Comment:
I do not think this solves the problem fully. "Your server did not start" message does not arrive from handle_pexpect function but from TypeError. I think the error can lie deeper than changing pexpect timeout. I gotta check that one